crafters/nlp/PDFExtractorSegmenter/tests/test_pdf_flow.py

```diff
@@ -22,26 +19,27 @@ def validate_img_fn(resp):
     for d in resp.search.docs:
         for chunk in range(len(d.chunks)):
             img = Image.open(os.path.join(cur_dir, f'test_img_{chunk}.jpg'))
-            blob = d.chunks[chunk].blob
+            blob = d.chunks[chunk].blob.dense
             assert blob.shape[1], blob.shape[0] == img.size
 
 
 def validate_mix_fn(resp):
     for d in resp.search.docs:
         for chunk in range(len(d.chunks) - 1):
             img = Image.open(os.path.join(cur_dir, f'test_img_{chunk}.jpg'))
-            blob = d.chunks[chunk].blob
+            blob = d.chunks[chunk].blob.dense
             assert blob.shape[1], blob.shape[0] == img.size
 
         assert expected_text == d.chunks[2].text
 
 
 def search_generator(path: str, buffer: bytes):
-    d = jina_pb2.Document()
+    d = Document()
+    d.update_id()
     if buffer:
         d.buffer = buffer
     if path:
-        d.uri = path
+        d.content = path
```

Comment:
I am not 100% sure the content is filling the 'uri'. Is 'uri' not exposed in current document type?